deploy/chart/templates/0000_90_olm_01-prometheus-rule.yaml

```diff
@@ -9,12 +9,23 @@ metadata:
     role: alert-rules
 spec:
   groups:
-    - name: olm.failing_operators.rules
+    - name: olm.csv_abnormal.rules
       rules:
-        - alert: FailingOperator
+        - alert: CsvAbnormalFailedOver2Min
+          expr: csv_abnormal{phase=~"^Failed$",exported_namespace=~"(^openshift.*|^kube.*|^redhat.*|^default$)"}
```

Comment:
the previous rule matched all namespaces, is there a reason we don't want to alert in other namespaces?

Reply:
This is in line with everything else in OpenShift, hence we did it here too. Do you have any concerns about it?

Reply:
It looks like these rules are gated on the install platform being OCP, and not a vanilla k8s cluster - is there any other reason besides we document that certain operators should be installed in certain prefixed (e.g. `openshift-*`) namespaces? If not, it seems fine to me to match against all namespaces in the cluster, vs. the redhat-controlled ones.